### ISSUE TITLE
Quantitative results on DTU

### ISSUE THREAD

**Issue by ghost** [RongLiu-Leo/AtomicGS#4]
Thanks for your code release!
I run the most recent version of AtomGS on DTU scans (data taken from [2DGS](https://github.com/hbb1/2d-gaussian-splatting)) with the following training script:
```
python train.py -s <input path> -m <output path> --scaling_lr 0 --iteration 7000
```
And the quality of the output meshes seems not so well as visualized in paper. For example, the Scan 24 mesh is noisy with a 1.21 evaluated CD value (rather than 0.51 claimed in paper).
![2024-06-03 203933](https://github.com/RongLiu-Leo/AtomGS/assets/171479623/58f8ddaf-beb4-4ad1-bb14-d8778a645670)
I tried to increase the `lambda_normal` value following your tips, but it didn't work. Could you please share the optimal experiment setting or give some instructions?

**Comment by RongLiu-Leo** [comment]
Yes, the default setting is for the Mip-NeRF 360 dataset, so the hyperparameter may need to be fine-tuned for other datasets.

Unfortunately, I lost the exact config for DTU results, but if I remember correctly, it should be
```
python train.py -s <input path> -m <output path> --scaling_lr 0 --lambda_ms_ssim 1 --atom_init_quantile 0.1 --iteration 7000
```
L1 loss will benefit the PSNR values but be noisier for geometry, so we used MS-SSIM loss instead.
A larger Atom Scale also helps reduce the Atom-shape noise.

**Comment by RongLiu-Leo** [comment]
Our pre-trained GS models and exported Poisson Mesh on DTU are available [here](https://drive.google.com/drive/folders/1hnuqFOjb4USPZyUzU4DpLHthLnMudRP4?usp=sharing).

One possible reason for our results is that we use the masked DTU dataset to filter out the background.

There is always a trade-off between achieving smooth surfaces and capturing high-frequency details. 2DGS provides a smoother surface but loses intricate details, such as those spokes in the bicycle scene. On the other hand, AtomGS enhances high-frequency details but introduces a certain level of noise.

One of our limitations is the use of a simple Poisson mesh extractor. The high-quality normal maps seen in the viewer cannot be perfectly extracted to mesh.

In the future, we aim to use a better extractor to export the refined geometry from the radiance field. Additionally, it is evident that 2DGS and AtomGS can complement each other by addressing their respective limitations, so we plan to integrate these two approaches.
